src/app/domains/Competence/components/views/CompetenceSimpleView/CompetenceSimpleView.template.js

```diff
@@ -1,15 +1,15 @@
 import PropTypes from 'prop-types'
-import { Tag, Typography } from 'antd'
+import { Typography } from 'antd'
 import firestore from '~/services/Firebase/firestore/index'
 import { useDocumentData } from 'react-firebase-hooks/firestore'
-import { COLLECTIONS } from 'app/constants'
-const { Text } = Typography
+import * as COLLECTIONS from '~/app/constants/collections'
```

Comment:
Is it work with `import { COLLECTIONS } from 'app/constants'`?

Reply:
yes